hr_timesheet_sheet_autodraft/__manifest__.py

```diff
@@ -2,25 +2,18 @@
 # License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
 
 {
-    'name': 'HR Timesheet Sheet Auto-draft',
-    'version': '12.0.1.0.0',
-    'category': 'Human Resources',
-    'website': 'https://github.com/OCA/timesheet',
-    'author':
-        'Brainbean Apps, '
-        'Odoo Community Association (OCA)',
-    'license': 'AGPL-3',
-    'installable': True,
-    'application': False,
-    'summary': (
-        'Automatically draft a Timesheet Sheet for every time entry that does'
-        ' not have a relevant Timesheet Sheet existing.'
+    "name": "HR Timesheet Sheet Auto-draft",
+    "version": "12.0.1.0.0",
+    "category": "Human Resources",
+    "website": "https://github.com/OCA/timesheet",
+    "author": "Brainbean Apps, " "Odoo Community Association (OCA)",
```

Comment:
This `" "` after-effect of auto-format should be removed manually, but not anyhow important

Reply:
I have removed the extra `" "`